### DISCUSSION TITLE
How can I use kornis with torch 1.1 ?

### DISCUSSION THREAD

**Discussion by MayuOshima** [kornia/kornia#891] [Questions] [answered]
For some reason, I can only use torch version 1.1. But the README of kornia says that it can only support torch version >=1.2.

So is there a method that I can use kornia with torch 1.1 ?

Thanks !

**Comment by edgarriba** [comment, marked as answer]
@MayuOshima Kornia doesn't require any specific pytorch version at running time. This means that you could install Kornia and later manually downgrade PyTorch. However, I do not recommend doing that since you might experience some bugs or directly incompatibilities with functions. May I ask what's the reason of using such old torch version and what Kornia functionalities you want to use ?
